ts-packages/web/src/app/(social)/threads/[id]/_components/header.tsx

```
@@ -31,6 +31,7 @@ export default function Header({ post_id }: { post_id: number }) {
 
   const space_id = post?.spaces[0]?.id;
 
+  const author_id = post?.author[0].id;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'author_id'. Consider to delete it or rename it.

'author_id' is defined here:


(lint/suspicious/noRedeclare)